config.go

```diff
@@ -81,7 +82,9 @@ func validateConfig(c *gabs.Container) error {
 			if _, ok := v.Data().(float64); !ok {
 				errors = multierror.Append(errors, fmt.Errorf("expected float64 value for %T, got: %v", k, v.Data()))
 			}
-		case "ssl_off":
+		case
+		    "ssl_off",
+		    "pass":
```

Comment:
This looks extra - isn't the passKey now set to "*"?

Reply:
Whoops, good catch!